lib/logging.js

```diff
@@ -0,0 +1,18 @@
+var winston = require("winston");
+var config = require("./config")
+
+var consoleLogger = new winston.transports.Console({"timestamp": true});
+var fileLogger = new winston.transports.File({"filename": config.log_file,
+    "timestamp": true
+});
+
+var logger = new winston.Logger({
+    "emitErrs": false,
```

Comment:
This says what should happen if the logger fails to log to the described location. At the moment I am just supressing the errors